tests/unit/adaptors/test_caching.py

```diff
@@ -67,18 +86,23 @@ def test_make_data_file_name(self, include_external_packages, expected):
                 ),
             },
             include_external_packages=include_external_packages,
+            exclude_type_checking_imports=exclude_type_checking_imports,
         )
 
         assert result == expected
 
     @pytest.mark.parametrize(
-        "include_external_packages, expected",
+        "include_external_packages, exclude_type_checking_imports, expected",
         (
-            (False, "hyphenated-package,underscore_package"),
-            (True, "hyphenated-package,underscore_package:external"),
```

Comment:
Got lazy to combine different parametrize together, I believe covering all the combinations manually is fine for now